ph5/utilities/segd2ph5.py

```diff
@@ -177,13 +177,27 @@ def get_args():
                        action='store_true', default=False)
 
     oparser.add_option("-M", "--num_mini", dest="num_mini",
-                       help="Create a given number of miniPH5 files.",
+                       help=("Create a given number of miniPH5 files. "
+                             "Ex: -M 38"),
                        metavar="num_mini", type='int', default=None)
 
     oparser.add_option("-S", "--first_mini", dest="first_mini",
-                       help="The index of the first miniPH5_xxxxx.ph5 file.",
+                       help=("The index of the first miniPH5_xxxxx.ph5 file "
+                             "of all. Ex: -S 5"),
                        metavar="first_mini", type='int', default=1)
 
+    oparser.add_option("-F", "--from_mini", dest="from_mini",
+                       help=("The index to continue miniPH5_xxxxx.ph5 file "
+                             "from. Do not associate with option -M. "
+                             "Ex: -F 25"),
+                       metavar="from_mini", type='int', default=None)
```

Comment:
metavar isn't necessary for named arguments if setting to the same as the arg name